Gemfile.lock

```diff
@@ -436,4 +436,4 @@ RUBY VERSION
    ruby 2.4.2p198
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
I think the Bundler version should be changed back from 2.0.1 to 1.17.3 because it might be causing errors otherwise?